client/src/App.js

```diff
@@ -19,7 +19,7 @@ import Login from "./pages/Login";
 import Signup from "./pages/Signup";
 import Profile from "./pages/Profile";
 import Publish from "./pages/Publish";
-import cloudinaryWidget from "./components/Cloudinary";
```

Comment:
removing this?

Reply:
probably don't need it, but not sure

Reply:
Lets keep anything cloudinary for now I guess since thats why my app isnt starting now